examples/usdConverter/README.md

```diff
@@ -0,0 +1,30 @@
+# Converting between SDF and USD
+
+This example shows how a world in a SDF file can be converted to [USD](https://graphics.pixar.com/usd/release/index.html).
+
+## Requirements
+
+You will need all of the dependencies for sdformat, along with the following additional dependencies:
+* USD: [installation instructions](https://github.com/PixarAnimationStudios/USD/blob/release/README.md#getting-and-building-the-code)
+* [ignition-common4](https://github.com/ignitionrobotics/ign-common)
+
+## Setup
+
+Build sdformat, and then run the following commands to build the example (run these commands from this example directory):
+```bash
+mkdir build
+cd build
+cmake ..
+make
+```
+
+You should now have an executable named `sdf2usd`, which can be used to convert a SDF world file to a USD file.
+The following command converts the example `shapes.sdf` file to its USD representation, stored in a file called `shapes.usda` (run this command from the `build` directory):
+```bash
+./sdf2usd ../shapes.sdf shapes.usda
+```
```

Comment:
The `sdf2usd` util has a different syntax than the example given.

```
Sdf format converter
Usage: ./sdf2usd [OPTIONS]

Options:
  -h,--help                   Print this help message and exit
  --help-all                  Show all help
  --version                   
  -i,--input TEXT             Input filename
  -o,--output TEXT            Output filename
```

It requires the `-i` and `-o` flag instead of positionals.